### PR TITLE
SDI-305 Additional tests and reformatting

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/service/VisitService.kt [hunk 1]
```diff
@@ -176,6 +176,21 @@ class VisitService(
     log.debug("Visit with Nomis visit id = ${visit.id} cancelled")
   }
 
+  fun getVisit(visitId: Long): VisitResponse {
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/service/VisitService.kt:179]: moved from bottom of file so public methods are at the top

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Repository.kt [hunk 2]
```diff
@@ -99,10 +112,16 @@ class Repository(
...
+  fun lookupAgencyInternalLocationByDescription(description: String): AgencyInternalLocation =
+    agencyInternalLocationRepository.findOneByDescription(description).map { it }.orElse(null)
+
   fun delete(offender: Offender) = offenderRepository.deleteById(offender.id)
   fun delete(people: Collection<Person>) = personRepository.deleteAllById(people.map { it.id })
 
+  fun deleteAllVisitSlots() = agencyVisitSlotRepository.deleteAll()
+  fun deleteAllVisitDays() = agencyVisitDayRepository.deleteAll()
+  fun deleteAllVisitTimes() = agencyVisitTimeRepository.deleteAll()
+
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Repository.kt:124]: seed data comes with no slots/times/days so ok to delete all of them

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/resource/VisitResourceIntTest.kt [hunk 3]
```diff
@@ -171,7 +174,6 @@ class VisitResourceIntTest : IntegrationTestBase() {
             "endTime"           : "13:04",
             "prisonId"          : "$prisonId",
             "visitorPersonIds"  : [$personIds],
-            "visitRoomId"       : "VISIT",
             "issueDate"         : "2021-11-02",
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/resource/VisitResourceIntTest.kt:177]: no longer present in request

**Review by steverendell — APPROVED** [review]
lgtm

**Pull request merged**